test/callbacks.js

```diff
@@ -61,8 +62,11 @@ test('reentrant with callbacks deferred', (t) => {
     t.notOk(thirdLoaded, 'third is not loaded')
     firstLoaded = true
     s.use(second, function () {
-      s.use(third, done)
+      t.throws(() => {
+        s.use(third)
+      }, 'Impossible to load (third) because the parent (first) already loaded')
```

Comment:
The error is:
`throw new Error('Impossible to load (${obj.name}) plugin because the parent (${current.name}) was already loaded')`
Why here there is not the `was`?